ConcernsCaseWork/ConcernsCaseWork/Services/Teams/ITeamsModelService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ConcernsCaseWork.Models.Teams;
+using System.Threading.Tasks;
+
+namespace ConcernsCaseWork.Services.Teams
+{
+	public interface ITeamsModelService
+	{
+		public Task<ConcernsTeamCaseworkModel> GetCaseworkTeam(string ownerId);
```

Comment:
Method names should have 'Async' suffix

Reply:
some do, some don't. We have agreed to address this with an editor config rule